catalog/core/catalog-core-localstorageprovider/src/test/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProviderTest.java

```diff
@@ -231,6 +279,129 @@ public void testDelete() throws Exception {
+                NITF_MIME_TYPE,
+                TEST_INPUT_FILENAME);
+        URI unqualifiedUri = new URI(createResponse.getCreatedContentItems()
+                .get(0)
+                .getUri());
+
+        createResponse = assertContentItemWithQualifier(TEST_INPUT_CONTENTS,
+                NITF_MIME_TYPE,
+                TEST_INPUT_FILENAME,
+                createResponse.getCreatedContentItems()
+                        .get(0)
+                        .getId(),
+                QUALIFIER);
+
+        URI qualifiedUri = new URI(createResponse.getCreatedContentItems()
+                .get(0)
+                .getUri());
+
+        String id = createResponse.getCreatedContentItems()
+                .get(0)
+                .getId();
+        ByteSource byteSource = new ByteSource() {
+            @Override
+            public InputStream openStream() throws IOException {
+                return IOUtils.toInputStream("Updated NITF");
+            }
+        };
+        ContentItem updateItem = new ContentItemImpl(id,
+                byteSource,
+                NITF_MIME_TYPE,
+                mock(Metacard.class));
+        UpdateStorageRequest updateRequest = new UpdateStorageRequestImpl(Collections.singletonList(
+                updateItem), null);
+
+        assertUpdateRequest(updateRequest);
+
+        updateItem = new ContentItemImpl(id,
+                qualifiedUri.getFragment(),
+                byteSource,
+                NITF_MIME_TYPE,
+                mock(Metacard.class));
+
+        updateRequest = new UpdateStorageRequestImpl(Collections.singletonList(updateItem), null);
+
+        assertUpdateRequest(updateRequest);
+    }
+
+    @Test
+    public void testDeleteIdWithMultpleContent() throws Exception {
+        CreateStorageResponse createResponse = assertContentItem(TEST_INPUT_CONTENTS,
+                NITF_MIME_TYPE,
+                TEST_INPUT_FILENAME);
+        createResponse = assertContentItemWithQualifier(TEST_INPUT_CONTENTS,
+                NITF_MIME_TYPE,
+                TEST_INPUT_FILENAME,
+                createResponse.getCreatedContentItems()
+                        .get(0)
+                        .getId(),
+                QUALIFIER);
+        String id = createResponse.getCreatedContentItems()
+                .get(0)
+                .getId();
+        DeleteStorageRequest deleteRequest =
+                new DeleteStorageRequestImpl(createResponse.getCreatedContentItems()
+                        .stream()
+                        .map(ContentItem::getMetacard)
+                        .collect(Collectors.toList()), null);
+        when(deleteRequest.getMetacards()
+                .get(0)
+                .getId()).thenReturn(id);
+
+        DeleteStorageResponse deleteResponse = provider.delete(deleteRequest);
+        provider.commit(deleteRequest);
+        List<ContentItem> items = deleteResponse.getDeletedContentItems();
+        assertThat(items.size(), is(2));
```

Comment:
Could use the [`hasSize()`](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html#hasSize(int)) matcher.